### PR TITLE
Andi bugs

### PR DESCRIPTION
**Pull request by cameronmarsh**

### CODE REVIEW

File: apps/andi/assets/js/app.js [hunk 1]
```diff
@@ -28,7 +28,7 @@ Hooks.showSnackbar = {
     updated() {
         let snackbar = document.getElementById("snackbar");
         snackbar.className += " show";
-        setTimeout(function() { snackbar.className = snackbar.className.replace("show", ""); }, 3000);
+        setTimeout(function() { snackbar.className = snackbar.className.replace(" show", ""); }, 3000);
```
Comment (bbrewer-forge) [inline apps/andi/assets/js/app.js:31]: oh

File: apps/andi/lib/andi_web/live/helpers/form_section.ex [hunk 2]
```diff
@@ -15,6 +15,7 @@ defmodule AndiWeb.FormSection do
           socket.assigns.changeset
           |> Map.put(:action, :update)
 
+        AndiWeb.Endpoint.broadcast_from(self(), "form-save", "save-all", %{})
```
Comment (bbrewer-forge) [inline apps/andi/lib/andi_web/live/helpers/form_section.ex:18]: Are these synchronous enough to not be a problem?

Comment (cameronmarsh) [reply apps/andi/lib/andi_web/live/helpers/form_section.ex:18]: I haven't run into any issues with it, but that's a good point

File: apps/andi/assets/js/app.js (hunk 1 above)
Comment (cameronmarsh) [reply apps/andi/assets/js/app.js:31]: :-|

**Pull request merged**